sw/device/lib/dif/dif_entropy.c

```diff
@@ -46,8 +44,7 @@ static void set_config_register(const dif_entropy_t *entropy,
                                rng_bit_sel);
 
   // Enable configuration
-  reg =
-      bitfield_field32_write(reg, ENTROPY_SRC_CONF_ENABLE_FIELD, config->mode);
+  reg = bitfield_field32_write(reg, ENTROPY_SRC_CONF_ENABLE_FIELD, 1);
   mmio_region_write32(entropy->params.base_addr, ENTROPY_SRC_CONF_REG_OFFSET,
```

Comment:
This will eventually be a field, but the config->mode can indeed be dropped since there is no LFSR mode.